test.rb

```diff
@@ -0,0 +1,24 @@
+load 'enumerable.rb'
+
+def multiply_els(arr)
+  arr.my_inject { |sum, n| sum * n }
+end
```

Comment:
### #multiply_els
* [ ] Accepts an array as an argument and multiplies all the elements of the array together using #my_inject

Reply:
please check the test.rb file for this test case